govcd/api_vcd_test.go

```diff
@@ -56,6 +56,7 @@ const (
 	TestRefreshOrgFullName        = "TestRefreshOrgFullName"
 	TestUpdateCatalog             = "TestUpdateCatalog"
 	TestDeleteCatalog             = "TestDeleteCatalog"
+	TestRenameCatalog             = "TestRenameCatalog"
```

Comment:
Instead of using a global constant, define a local variable using `check.Testname()` within the test

Reply:
Fixed.